### DISCUSSION TITLE
How to hide ->reorderable('sort') if table is empty?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#12411] [Help] [answered]
### Package

Table builder

### Package Version

v3.2.70

### How can we help you?

I use `->reorderable($this->table->getQuery()->count() > 0 ? null : 'sort')`

but I get this error

**Typed property App\Livewire\DesignsUpload\DesignsList::$table must not be accessed before initialization**

**Comment by yuvrajtimalsina** [comment, marked as answer]
**Try this :**

```
->reorderable('sort', $table->getQuery()->count() > 0)
```
